### ISSUE TITLE
[react-icons] убрать все использования react-icons

### ISSUE THREAD

**Issue by rustho** [skbkontur/retail-ui#2171]
Убрать зависимость от react-icons, всё равно они не используются нигде, кроме stories

**Comment by dzekh** [comment]
Вроде уже должны были убрать давно
@zhzz можешь проверить и переоткрыть, если вдруг еще актуально?

**Comment by zhzz** [comment]
Они все еще используются в историях.

**Comment by JackUait** [comment]
В историях и в доке. А мы вообще сможем от них когда-нибудь отказаться?

**Comment by dzekh** [comment]
Да, и есть ли необходимость в этом? В чем проблема, что есть зависимость от react-icons?
